common/on_action/wc_plague_on_actions.txt

```diff
@@ -0,0 +1,132 @@
+﻿# PLAGUE ON ACTIONS
```

Comment:
Could you show the plan for purge events?

Reply:
There are 2 files named the same. Need to change it.

Reply:
Purge events are divided in 3 categories: starting, purge, ending. Starting events are the ones that are related to getting inside the plague barony (something like finding an Ogre camp or a savage beast), purge events are the ones related to the purgiong activity (killing undead, burning stuff...) and the ending is a finale, like, fighting some boss (abomination, lich). You have to suceed in some of the events in each stage to get to the next one.
Also, if a war happens, the "starting events" won't be needed because you are sort of already there.

Reply:
Cleaned up, there's no files with same name anymore.

Reply:
> Cleaned up, there's no files with same name anymore.

Not these files, they can't be merged, they're from different folders.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132714510-41d8f64f-06e0-44a5-a616-598e298402c9.png)

</details>

I meant the on action files, `wc_plague_on_actions.txt`.

Reply:
> fighting some boss (abomination, lich)

Hmm? Who fights? If it's just one character, what is army doing at that moment?